fdbserver/include/fdbserver/ShardsAffectedByTeamFailure.h

```diff
@@ -36,7 +36,9 @@ class ShardsAffectedByTeamFailure : public ReferenceCounted<ShardsAffectedByTeam
 		bool primary;
 
 		Team() : primary(true) {}
-		Team(std::vector<UID> const& servers, bool primary) : servers(servers), primary(primary) {}
+		Team(std::vector<UID> const& servers, bool primary) : servers(servers), primary(primary) {
+			ASSERT(std::is_sorted(servers.begin(), servers.end()));
```

Comment:
Why not sort inside the constructor? Is this a performance optimization?

Reply:
Yeah, not sure though an extra sort has much additional cost given the `servers` is often short. But it seems like our old code using `Team` was already sorted before creating a `Team` object. And there are other places not just in `Team` assuming the `vector<UID>` is sorted. So if I sort `Team.servers` here, another sort is also needed outside for the parameter `servers`.